ruma-identity-service-api/src/tos/get_terms_of_service/v2.rs

```diff
@@ -0,0 +1,62 @@
+//! [GET /_matrix/identity/v2/terms](https://matrix.org/docs/spec/identity_service/r0.3.0#get-matrix-identity-v2-terms)
+
+use std::collections::BTreeMap;
+
+use ruma_api::ruma_api;
+use serde::{Deserialize, Serialize};
+
+ruma_api! {
+        metadata: {
+            description: "Gets all the terms of service offered by the server. The client is expected to filter through the terms to determine which terms need acceptance from the user. Note that this endpoint does not require authentication.",
+            method: GET,
+            name: "tos::get_terms_of_service::v2",
+            path: "/_matrix/identity/v2/terms",
+            authentication: None,
+            rate_limited: false,
+        }
+
+        #[derive(Default)]
+        request: {}
+
+        response: {
+            /// Required. The policies the server offers. Mapped from arbitrary ID
+            /// (unused in this version of the specification) to a Policy Object.
+            pub policies: BTreeMap<String, Policies>
+        }
+}
+
+impl Request {
+    /// Creates an empty `Request`.
+    pub fn new() -> Self {
+        Self
+    }
+}
+
+impl Response {
+    /// Creates a new `Response` with the given `Policies`
+    pub fn new(policies: BTreeMap<String, Policies>) -> Self {
+        Self { policies }
+    }
+}
+
+/// Collection of localized policies
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct Policies {
+    /// Required. The version for the policy. There are no requirements on what
+    /// this might be and could be "alpha", semantically versioned, or arbitrary.
+    pub version: String,
+
+    /// Optional. Available languages for the policy.
```

Comment:
A hint at what the `String` in the type below means (with examples) would be good.

Reply:
Changes made. I have added examples for all the fields.

Reply:
I've also removed the "Optional". I thought it didn't make much sense to have it anymore, considering that it would be pretty much pointless to send a `Policies` object. I'm not sure, and the spec isn't helpful here.